### ISSUE TITLE
The hard links from backup_source are not copied to backup_destination

### ISSUE THREAD

**Issue by Bakaras** [ugoviti/synbak#3]
When i run the synbak with rsync then hardlinks are lost.

```
On source host:
root@n21:~# ls -li /usr/bin/zipinfo
136721 -rwxr-xr-x 2 root root 158392 Dez 13  2016 /usr/bin/zipinfo

root@n21:~# find / -xdev -inum 136721
/usr/bin/zipinfo
/usr/bin/unzip

root@n21:~# ls -li /usr/bin/unzip
136721 -rwxr-xr-x 2 root root 158392 Dez 13  2016 /usr/bin/unzip
```
On synbak machine:
```
root@synbak:/synbak/n21/rsync/daily/backup-n21-20180725-113441# ls -li usr/bin/zipinfo
18366364 -rwxr-xr-x 1 root root 158392 Dez 13  2016 usr/bin/zipinfo

root@synbak:/synbak/n21/rsync/daily/backup-n21-20180725-113441# find . -xdev -inum 18366364
./usr/bin/zipinfo

root@synbak:/synbak/n21/rsync/daily/backup-n21-20180725-113441# ls -li usr/bin/unzip
18366319 -rwxr-xr-x 1 root root 158392 Dez 13  2016 usr/bin/unzip
```

**Comment by ugoviti** [comment]
Hi, you are right... Thanks for the bug report, I'll add the -H option in the next release... 

Right now you can workaround adding:

backup_method_opts = -H

Let's me know if it doesn't works.

King regards

**Comment by ugoviti** [comment]
fixed in 3.7 release, thank you!
